build-and-test.ps1

```diff
@@ -3,12 +3,29 @@ param(
     [switch]$UseImageCache,
     [string]$VersionFilter,
     [string]$ArchitectureFilter,
-    [string]$OSFilter
+    [string]$OSFilter,
+    [switch]$CleanupDocker
 )
 
 Set-StrictMode -Version Latest
 $ErrorActionPreference = 'Stop'
 
+function Invoke-CleanupDocker()
+{
+    if ($CleanupDocker)
+    {
+        docker ps -a -q | ForEach-Object { docker rm -f $_ }
+        docker images |
```

Comment:
I think a comment would be justified on why the base Windows OS images are left.